src/components/orderDetails/OrderDetails.tsx

```diff
@@ -0,0 +1,345 @@
+            {ORDER_DETAILS}
+          </span>
+          <span
+            className="w-1/2 justify-end flex items-center !text-[12px] font-normal cursor-pointer !bg-[#1C1C20] sm:!bg-inherit"
+            onClick={showPromotions}
+          >
+            <CustomIcon
+              className="mr-1"
+              alt="promotion-icon"
+              src={PromotionsIcon}
+              width="0.75rem"
+              height="0.75rem"
+            />
+            {PROMOTIONS}
+          </span>
+        </div>
+        <div className="flex flex-col border-t-0 sm:border-t border-solid border-[#383F47] sm:grid sm:gap-y-0 sm:grid-cols-2 ">
+          <div className="flexWrapper justify-start bg-[#30343B] rounded-t-md">
+            <span className="w-auto sm:w-1/5 flex items-center">
+              {ORDER}
+              <CustomIcon
+                className="ml-2 cursor-pointer"
+                alt="order-clock-icon"
+                src={OrderClockIcon}
+                width="1rem"
+                height="1rem"
+                onClick={showOrderTimeline}
+              />
+            </span>
+            <span className="w-auto sm:w-4/5 font-medium">
+              {orderData?.orderId}
+            </span>
+          </div>
+          <div className="flexWrapper bg-[#30343B]">
+            <span className="w-auto sm:w-1/5 font-light">{ORDER_TOTAL}</span>
+            <span className="w-auto sm:w-4/5 font-medium">
+              {orderData?.orderTotal}
+            </span>
+          </div>
+        </div>
+        <div className="flex flex-col border-t-0 sm:border-t border-solid border-[#383F47] sm:grid sm:gap-y-0 sm:grid-cols-2 bg-[#30343B]">
+          <div className="flexWrapper">
+            <span className="w-auto sm:w-1/5 font-light">{SUBMITTED}</span>
+            <span className="w-auto sm:w-4/5 font-medium">
+              {orderData?.submittedDate}
+            </span>
+          </div>
+          <div className="flexWrapper">
+            <span className="w-auto sm:w-1/5 font-light">{CHANNEL}</span>
+            <span className="w-auto sm:w-4/5 font-medium">
+              {orderData?.originOfOrder}
+            </span>
+          </div>
+        </div>
+        <div className="flex flex-col border-t-0 sm:border-t border-solid border-[#383F47] sm:grid sm:gap-y-0 sm:grid-cols-2 bg-[#30343B] rounded-b-md">
+          <div className="flexWrapper">
+            <span className="w-auto sm:w-1/5 font-light">{LOCALE}</span>
+            <span className="w-auto sm:w-4/5 font-medium">
+              {orderData?.locale}
+            </span>
+          </div>
+          <div className="flex justify-between w-full py-1 px-4 sm:p-0 border-none">
+            <span className="w-auto sm:w-1/5">{ORDER_TYPE}</span>
+            <span className="w-auto sm:w-4/5 font-medium">
+              {orderData?.customerInfo?.biTier}
+            </span>
+          </div>
+        </div>
+      </div>
+
```

Comment:
I can still see some more classes being repeated again and again.
Please check the whole component and make similar changes.

Reply:
Done